### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by mario-bucev**
For 0.9.7

### CODE REVIEW

**Comment by vkuncak** [comment]
We should fix the docs:
  * the output shown here: https://github.com/epfl-lara/stainless/blob/main/core/src/sphinx/gettingstarted.rst
  * the JDK and other instructions here: https://epfl-lara.github.io/stainless/installation.html

**Review by vkuncak — APPROVED** [review]

**Comment by vkuncak** [comment]
Well, a release tag should ideally contain sphinx sources that are up to
date documentation for the current version. gh-paged affects what is
rendered as the current documentation.

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Release Notes
 
+## Version 0.9.7 (2022-11-21)
+
```
[the remark this answers is not in the document]
Comment (mario-bucev) [reply docs/RELEASE_NOTES.md:4]: Will do on the GitHub release

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+### Stainless frontend, library and internals
+
+- Improve equivalence checking: function call matching, norm, mkTest (#1294)
+- Experimental integration of OL- and OCBSL- based simplifiers (#1315)
+- Upgrade to Scala 3.2 (#1317)
+- Add verification pipeline summary (#1336)
+- Fix issues #1332, #1271, #1333, #731, #1290, #1321, #1322, #1306, #1301, #1302
```
[the remark this answers is not in the document]
Comment (mario-bucev) [reply docs/RELEASE_NOTES.md:11]: Ditto :)

**Pull request merged**